src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private static final int MAX_POSSIBLE_NUMBER = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int randomInteger = new Random().nextInt(MAX_POSSIBLE_NUMBER + 1);
+        return new Ball(randomInteger, colorSupplier.getRandomColor());

```

Comment:
make ColorSupplier and Random class-fields

Reply:
please give some explanation why I should do so. For better readability ar something else?